watchdog.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
yeah we should clean this up... and make a sane pattern.
When we do the next cleanup iteration of all component, move to conf, glue code, hapi and other things we have pending...